examples/gradle/dokka-versioning-multimodule-example/parentProject/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+dependencies {
+    implementation(kotlin("stdlib"))
+}
+
+tasks.dokkaHtmlMultiModule {
```

Comment:
i think that it would be nice to have a task that depends on another 2 tasks which generate previous versions, so user can see how expected workflow looks like.